src/React/CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## 2.28.0
+
+-   [BC BREAK] If you are using the Symfony AssetMapper but **not** Symfony Flex,
+    you need to upgrade your `importmap.php` and change the asset `react-dom` to `react-dom/client`,
+    and run `php bin/console importmap:install`.
+
+    Symfony Flex or Webpack Encore users are not affected.
```

Comment:
I really don't see why we would release this. No feature, no bug, but a big break in userland code.

Let's at least wait to push until we have something to....

Reply:
> Ok I was not yet here when i commented previous comment :)
> 
> Thanks for the detailed explanation... I still think that releasing a version that will break people apps, when we bring zero new feature, is neither a good habit nor a good message to send 🤷

Yeah I agree with you

> But i understand you want to change everything at once... So to me we need to find a way to "not" release React here.
> 
> WDYT ?

I understand, but I don't think that's good to maintain two similar but different building system. 

... What about releasing it for UX 3 ? :D

Reply:
I agree with you on everything... i just cannot stress enough that a change in build is not a feature. 

And maintain npm only package is not something i refuse or don't want (they would still work with importmap :) )... 

... i really think without a way to synchronize releases for both php and js vendor... and this is almost impossible to do properly 🤷